### PR TITLE
PayPal button component and Credit Card button component

### LINKED ISSUE #158: Turn payment option buttons into components
- [ ] PapPal
- [ ] credit card

### PR DESCRIPTION
**Pull request by mmmavis**
This fixes #158

### CODE REVIEW

File: components/credit-card-button.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { FormattedHTMLMessage, IntlMixin } from 'react-intl';
```
Comment (alicoding) [inline components/credit-card-button.jsx]: You don't need to import both here.

File: components/credit-card-button.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+var CreditCardButton = React.createClass({
+  mixins: [IntlMixin],
```
Comment (alicoding) [inline components/credit-card-button.jsx]: Just do `require('react-intl').IntlMixin`

File: components/paypal-button.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { FormattedHTMLMessage, IntlMixin } from 'react-intl';
```
Comment (alicoding) [inline components/paypal-button.jsx]: same here.

**Comment by mmmavis** [comment]
@alicoding review?

**Pull request merged**